contracts/community-nft1.0.sol

```diff
@@ -0,0 +1,297 @@
+pragma solidity ^0.6.2;
+pragma experimental ABIEncoderV2;
```

Comment:
If you use the latest solidity version, e.g. ^0.8.0, the ABIEncoderV2 is not experimental anymore.